docs/src/api/class-locatorassertions.md

```diff
@@ -495,6 +507,12 @@ assertThat(page.locator("list > .component")).hasText(new String[] {"Text 1", "T
 ```
 
 ### param: LocatorAssertions.toHaveText.expected
+- `expected` <[string]|[RegExp]|[Array]<[string]|[RegExp]>>
```

Comment:
Don't you need to specify language filter in both overrides?

Reply:
added.